src/main/steps/edge-case/upload-appeal-form/template.njk

```diff
@@ -32,9 +32,9 @@
     {%if uploadedDocuments.length >0 %}
     {% for file in uploadedDocuments %}
       <li class="uploadedFile govuk-!-padding-top-2 govuk-!-padding-bottom-3 govuk-section-break govuk-section-break--visible">
-        {{ file.fileName }}
+        <p class="govuk-body" id="Document{{ loop.index0 }}">{{ file.fileName }}</p>
         {% set documentId = file.documentId%}
-        <a href="/upload-appeal-form?query=delete&docId={{documentId}}&documentType=tribunalform" class="govuk-link--no-visited-state float-right ">{{ delete }}</a>
+        <a href="/upload-appeal-form?query=delete&docId={{documentId}}&documentType=tribunalform" class="govuk-link--no-visited-state float-right " id="Delete{{ loop.index0 }}" aria-labelledby="Delete{{ loop.index0 }} Document{{ loop.index0 }}">{{ delete }}</a>
```

Comment:
Changes for these could have been made in assets/js/upload-manager/updateFileList.ts but it was not being used. I think it used for uploading to /document-manager (DOCUMENT_UPLOAD_URL) but the code doesn't currently do that at any point.